tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -188,7 +190,9 @@
                     "members"
                 ],
                 "additionalProperties": false
-            }
+            },
+	    "minItems": 1,
```

Comment:
For groups, don't we want to allow the tool to have none of these?

Reply:
yes we do, in this case there won't be any `groups` property. But if the property is there, the array can't be empty.

Reply:
Great, thanks! 😄